src/dist/temp.rs

```diff
@@ -4,21 +4,21 @@ use std::io;
 use std::ops;
 use std::path::{Path, PathBuf};
 
-pub use anyhow::{Context, Result};
+pub(crate) use anyhow::{Context, Result};
 use thiserror::Error as ThisError;
 
 use crate::utils::notify::NotificationLevel;
 use crate::utils::raw;
 use crate::utils::utils;
 
 #[derive(Debug, ThisError)]
-pub enum Error {
+pub(crate) enum CreatingError {
     #[error("could not create temp root {}" ,.0.display())]
-    CreatingRoot(PathBuf),
+    Root(PathBuf),
     #[error("could not create temp file {}",.0.display())]
-    CreatingFile(PathBuf),
+    File(PathBuf),
     #[error("could not create temp directory {}",.0.display())]
-    CreatingDirectory(PathBuf),
+    Directory(PathBuf),
 }
```

Comment:
I do not like having this name change conflated with a `pub(crate)` refactor.  Also I'm not convinced by it anyway.  The variant names now do not read intuitively.  `CreatingError::Root` has the concepts of 'create' and 'error' reversed in an english grammatical sense.  `Error::CreatingRoot` makes much more sense.

Reply:
I changed it because I saw clippy's warning when I modified the code. Please see: https://rust-lang.github.io/rust-clippy/master/#enum_variant_names